### PR TITLE
CSCQVAIN-142: Allow selecting organizations from reference data

### PR DESCRIPTION
**Pull request by tahme**

### CODE REVIEW

File: src/components/ReferenceData.vue [hunk 1]
```diff
@@ -208,18 +245,19 @@ export default {
...
 			this.isLoading = false
 		},
 		async searchReferenceData(searchQuery) {
 			this.isLoading = true
-			const res = await esApiSearchClient(this.esIndex, this.esDoctype, searchQuery, this.count)
+			const res = await esApiSearchClient(this.esIndex, this.esDoctype, searchQuery + this.esQueryExtra, this.count)
 			this.responseData = res.data
 			this.isLoading = false
 		},
 		// TODO: if the es server is under too much stress debounce could be implemented
```
Comment (jppiiroinen) [inline src/components/ReferenceData.vue]: this should be removed at this time as well

File: src/components/Organization.vue [hunk 2]
```diff
@@ -0,0 +1,469 @@
...
+			</p>
+
+			<div
+				v-for="(org, i) in flattened"
+				:key="getKey(i)"
+				class="mb-2"
+			>
+				<div class="refrow">
+					<div class="org-label">
+						{{ new Array(i*4+1).join("&nbsp;&nbsp;") }}
```
Comment (jppiiroinen) [inline src/components/Organization.vue]: i wonder if this will leak

File: src/components/Organization.vue [hunk 3]
```diff
@@ -0,0 +1,469 @@
...
+			if (!this.value) { return -1 }
+			let recurse = this.value
+			let depth = 0
+			while (this.refField in recurse) {
+				depth++
+				recurse = recurse[this.refField]
+			}
+			return depth + 1
+		},
+		flattened() {
```
Comment (jppiiroinen) [inline src/components/Organization.vue:464]: should we have also a computed value for `flattened.length-1` which would be used to find the index

**Review by jppiiroinen — CHANGES_REQUESTED** [review]
have not tested yet, but did a quick review

File: src/components/Organization.vue (hunk 2 above)
Comment (tahme) [reply src/components/Organization.vue]: I had already replaced this with a better solution (found out about "string".repeat function) but forgot to push it

File: src/components/ReferenceData.vue (hunk 1 above)
Comment (tahme) [reply src/components/ReferenceData.vue]: not seeing any other comments about removing something, did you forgot to post some comment?

Comment (tahme) [reply src/components/ReferenceData.vue]: oh, did you mean to delete this comment line?

File: src/components/Organization.vue (hunk 3 above)
Comment (tahme) [reply src/components/Organization.vue:464]: done

File: src/components/ReferenceData.vue (hunk 1 above)
Comment (tahme) [reply src/components/ReferenceData.vue]: removed comment line

**Review by jppiiroinen — APPROVED** [review]

**Comment by tahme** [comment]
test comment

**Pull request closed without merging**